ci/jenkins/jobs/projects.yaml

```diff
@@ -410,6 +410,92 @@
           started_status: null
           wrappers: []
           publishers: []
+        - '{name}-{test_name}-for-pull-request':
+          test_name: full-features-conformance-pending-label
```

Comment:
sorry for the late comment, but I feel like `all-features-*` is a better name

Reply:
Updated.